packages/storybook/stories/AdhocCalling.stories.mdx

```diff
@@ -12,37 +13,44 @@ import AdhocCallingSnippetText from '!!raw-loader!./snippets/AdhocCalling.snippe
 
 # Adhoc calling
```

Comment:
I should have done it a long time ago but could you change all instances of 'adhoc' to 'ad-hoc' in this doc?